test/integration/scheduler/queue_test.go

```diff
@@ -1345,6 +1353,694 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod2"),
 			enableSchedulingQueueHint: []bool{true},
 		},
+		{
```

Comment:
This made the test to reach the 10 mins timeout https://github.com/kubernetes/kubernetes/issues/128660

We recommend to shard the test in different folders, go test build an independent test binary per folder so you have 10mins per tested folder